baseline.yaml

```
@@ -649,6 +649,38 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-18   
+    maturity_level: 2
+    category: Documentation
+    criteria: |
+       The project MUST perform threat modeling and
+       attack surface analysis to understand and protect
+       against critical code paths, functions, and interactions 
+       with the system.
```

```suggestion
       against attacks on critical code paths, functions, and interactions 
       within the system.
```
Presumably we don't want to prevent the use of critical code paths :-).